tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Features/Catalogue Solutions Update.feature

```diff
@@ -1,49 +1,87 @@
-﻿Feature: Update the catalogue solutions for an order
+﻿Feature: Update the catalogue solution order item for an order
     As a Buyer User
-    I want to update the catalogue solutions for a given order
-    So that I can keep an order up to date
+    I want to update the catalogue solution order item for a given order
+    So that I can keep my order up to date
 
 Background:
     Given Orders exist
-        | OrderId    | Description         | OrderStatusId | Created    | LastUpdated | LastUpdatedBy                        | OrganisationId                       |
-        | C000014-01 | Some Description    | 1             | 11/05/2020 | 11/05/2020  | 335392e4-4bb1-413b-9de5-36a85c9c0422 | 4af62b99-638c-4247-875e-965239cd0c48 |
+        | OrderId    | Description      | LastUpdatedBy                        | LastUpdatedByName | OrganisationId                       |
+        | C000014-01 | Some Description | 335392e4-4bb1-413b-9de5-36a85c9c0422 | Tom Smith         | 4af62b99-638c-4247-875e-965239cd0c48 |
+    And Service Recipients exist
+        | OdsCode | Name                | OrderId    |
+        | ODS1    | Service Recipient   | C000014-01 |
+        | ODS2    | Service Recipient 2 | C000014-01 |
+    And Order items exist
+        | OrderId    | OdsCode | CatalogueItemName | CatalogueItemType |
+        | C000014-01 | ODS1    | Order Item 1      | Solution          |
+        | C000014-01 | ODS2    | Order Item 2      | AdditionalService |
     And the user is logged in with the Buyer role for organisation 4af62b99-638c-4247-875e-965239cd0c48
 
-@4631
-Scenario: 1. Update an orders catalogue solutions
-    When the user makes a request to update the order catalogue solutions section with the ID C000014-01
+@7859
```

Comment:
I thought we were still using story numbers rather than task numbers? Could be wrong though